jenkins/e2e-image/e2e-runner.sh

```diff
@@ -432,6 +432,12 @@ elif [[ ! "${JOB_NAME}" =~ -pull- ]]; then
   fi
 fi
 
+if [[ -n "${PRIORITY_PATH:-}" ]]; then
+  export PATH="${PRIORITY_PATH}:${PATH}"
+fi
+echo "kubectl on path is:"
+kubectl version --client
```

Comment:
I'd rather we add `--print_client_version` to `e2e.go` or something, but I don't think it's a very big deal. @ixdy has been working on `kubectl version` I think, might want to weigh in.